Sources/SwiftLayout/Core/Builder/LayoutBuilder.swift

```diff
@@ -34,27 +34,39 @@ extension LayoutBuilder {
     public static func buildBlock<L: Layout, L1: Layout>(_ l: L, _ l1: L1) -> TupleLayout {
         TupleLayout(l, l1)
     }
-
+    
```

Comment:
Please remove unnecessary whitespace

Reply:
Originally, there was a whitespace, but it seems to have been caused by the code alignment operation
I opened the file and it seems that the blanking was handled as intended
Please check it again

Reply:
@gmlwhdtjd funny, @della-padula is right. may be github did something...?

Reply:
currently, every white space only lines has white spaces... 😂

Reply:
i do not checked trimming option in xcode settings. starting clean #251

Reply:
@della-padula @gmlwhdtjd so trimming "including whitespace-only line" now our convention?

Reply:
@oozoofrog Yes~ I think it would be nice to set it as a convention from now on~

Reply:
@oozoofrog @della-padula Good!! I totally agree!

I will resolve the conversation as it has nothing to do with this pr.
We'll cover the whitespace-related part in #251